CMakeLists.txt

```diff
@@ -17,7 +17,11 @@
 # Some variables like MSVC are defined only after project(), so put that first.
 project(diskann)
 
-cmake_minimum_required(VERSION 3.15)
+if(MSVC)
```

Comment:
I say we just make this a minimum of 3.15 across the board. Why use such an old version of cmake?

Reply:
I agree. I will just simplify this to make it 3.15 everywhere.